proto/cmp/types/v2/email.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package cmp.types.v2;
+
+// Email type for general use.
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1/email.proto.dot.xs.svg)
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1/email.proto.dot.svg)
+message Email {
+  string address = 1;
+
+  EmailType type = 2;
+
+  string description = 3;
```

Comment:
Is a description necessary when we have a lot of types to describe the address already?
Reason for that is that free text descriptions are something which has to be parsed to do something automatically with it if it's not for display purposes only.